docs/guides/connections.md

```diff
@@ -109,9 +109,10 @@ A Snowflake connection should be configured as follows:
 connections:
     my_snowflake_connection:
         type: snowflake
-        user: [username]
-        password: [password]
-        account: [account ID]
+        user: [required, username]
```

Comment:
Is this always required? Doesn't seem to match the field type in https://github.com/TobikoData/sqlmesh/commit/1220e602cea573479e9c70b792bc6a0614415ab4#diff-95f5a476d8642ba1937252ee40000b28f73c881331613e66abd63dac92514416R105.

cc: @eakmanrq

Reply:
I believe it is required if doing externalbrowser or basic auth but not required for Okta. So there is some nuance here. Maybe just update to say "required if not using Okta"?